### DISCUSSION TITLE
Where does the source code for `File Information` live?

### DISCUSSION THREAD

**Discussion by hirehamir** [microsoft/vscode-discussions#1171] [Extension Development QnA] [answered]
Also, how do I make that information show up for every file?

![image](https://github.com/microsoft/vscode-discussions/assets/43425812/024e1226-236a-4452-ba87-0a3a3b1f170d)

**Comment by hirehamir** [comment]
It'd be nice to see it across other file types, too. It doesn't show the file size for `.md` files, for instance.

![image](https://github.com/microsoft/vscode-discussions/assets/43425812/0d1e3506-42ef-431e-b463-5a199b5d61de)

**Comment by abrilurena** [comment]
Hi, @hamirmahal! do you still need help with this?

**Reply by hirehamir** [reply]
Hello, @abrilurena! Yes.

**Comment by isidorn** [comment]
I do not really understand the question here.
I will go ahead and close this, and feel free to open a new discussion with clear instructions. Thanks!

**Reply by hirehamir** [reply]
You're welcome. There's source code for the `File information` functionality, right? Where can I find it?

**Reply by isidorn** [reply, marked as answer]
Just do full text search for that displayed string in the VS Code repo and you should be able to find it.
